src/main/java/io/lakefs/routerfs/dto/FileSystemPathTuple.java

```diff
@@ -0,0 +1,16 @@
+package io.lakefs.routerfs.dto;
+
+import lombok.Getter;
+import org.apache.hadoop.fs.FileSystem;
+import org.apache.hadoop.fs.Path;
+
+@Getter
```

Comment:
[@Value](https://projectlombok.org/features/Value)?  (You could also get value types from other projects if you don't like this specific Value type...).